.github/workflows/examples.yml

```diff
@@ -79,6 +79,10 @@ jobs:
       shell: bash -l {0}
       run: |
         python examples/misc/linalg.py mat-vec mat-mat-sum transpose-mat-vec
+        python examples/misc/linalg.py mat-mat-sum --optimize
+        python examples/misc/linalg.py mat-mat --optimize
+        python examples/misc/linalg.py mat-vec --optimize
+        python examples/misc/linalg.py transpose-mat-vec --optimize
```

Comment:
what does this do again?

Reply:
Checks the linalg with `advanced`, it was `noop` only before.